packages/mdc-textfield/icon/_mixins.scss

```diff
@@ -42,7 +42,7 @@
     }
   } @else {
     .mdc-text-field__input {
-      @include mdc-rtl-reflexive-property(padding, $position /* left-value */, $padding  /* right-value */);
+      @include mdc-rtl-reflexive-property(padding, $input-padding /* left-value */, $padding  /* right-value */);
```

Comment:
I'm a little bit confused as to why this change is only needed within the `else`?

Reply:
Used `$input-padding` to `left` position too to make it semantically correct. Icon's `position` value and text field's `input-padding` are same anyway.